src/main/java/org/sonarlint/intellij/util/ProjectUtils.java

```diff
@@ -39,6 +37,8 @@
 import javax.annotation.CheckForNull;
 import org.sonarlint.intellij.finding.TextRangeMatcher;
 
+import static org.sonarlint.intellij.common.ui.ReadActionUtils.computeReadActionSafely;
```

Comment:
Is this added import used?

Reply:
Yes, it's just because the order of the static import moved to the bottom